plugins/wordpress-rule-exclusions-before.conf

```diff
@@ -708,14 +708,16 @@ SecRule REQUEST_FILENAME "@endsWith /wp-admin/nav-menus.php" \
     SecRule ARGS:action "@rx ^(?:update|edit)$" \
         "t:none,\
         chain"
-        SecRule &ARGS:action "@eq 1" \
+        SecRule &ARGS:action "@le 2" \
```

Comment:
Didn't `@eq 1` mean: "if there's an `action` arg"? Wouldn't the equivalent rather be `@ge 1` (at least 1)?

Reply:
Yes, it means if there's one instance of `action`. The false positive happens when `action` is sent twice (One as a query string, another in the body) so I updated the rule to check for `@le 2` two or less rather than one or more. I'd rather remove the chained rule for extra performance if we're going to do `@ge 1` one or more, at that point I don't see any benefit to the chained rule.

Reply:
Sure.

Reply:
Sure as in leave as is or something else?

Reply:
Sorry: go ahead and remove it if you feel the chain rule isn't necessary.

Reply:
I'll leave it as is for now.